permission_handler_platform_interface/lib/src/method_channel/method_channel_permission_handler.dart

```diff
@@ -72,7 +72,7 @@ class MethodChannelPermissionHandler extends PermissionHandlerPlatform {
   ///
   /// This method is only implemented on Android, calling this on iOS always
   /// returns [false].
-  Future<bool> shouldShowRequestPermissionRationale(
+  Future<bool?> shouldShowRequestPermissionRationale(
```

Comment:
Same as with the `openAppSettings` method. This method should never return `null` and should be handled here.